test/Microsoft.NET.Publish.Tests/GivenThatWeWantToCacheAProjectWithDependencies.cs

```diff
@@ -11,55 +11,93 @@
 using static Microsoft.NET.TestFramework.Commands.MSBuildTest;
 using System.Runtime.InteropServices;
 using Microsoft.DotNet.InternalAbstractions;
+using PackageInfoHelpers;
+using FluentAssertions;
+
 namespace Microsoft.NET.Publish.Tests
 {
     public class GivenThatWeWantToCacheAProjectWithDependencies : SdkTest
     {
+        private static string libPrefix;
+        private static string runtimeos;
+        private static string runtimelibos;
+        private static string runtimerid;
+        private static string testarch;
+        private static string tfm = "netcoreapp1.0";
+
+        static GivenThatWeWantToCacheAProjectWithDependencies()
```

Comment:
do we have a test that validates that the filtered packages are coming from the cache once the app targeting the cache is activated?